backend/server.js

```diff
@@ -34,7 +36,19 @@ if (config.get('NODE_ENV') !== 'production') {
   app.get('/graphiql', graphiqlExpress({ endpointURL: '/graphql' }))
 }
 
-app.use('/static', express.static(path.join(__dirname, 'images')))
-router.post('/static/upload', imageUploader) // TODO: Require Authentication
+function uploadAuth(req, res, next) {
```

Comment:
this middleware function feels odd defined here, maybe move it to `backend/middleware` or, if you're changing the `upload.js` file to use Router (see other comment), into there?